build.gradle

```diff
@@ -247,7 +247,7 @@ dependencies {
 
   implementation group: 'org.elasticsearch', name: 'elasticsearch', version: '8.9.1'
   implementation group: 'org.springframework.boot', name: 'spring-boot-devtools', version: '2.7.15'
-  implementation 'org.json:json:20230618'
+  implementation 'org.json:json:20230227'
```

Comment:
Do you want to add something to dependabot config to fix to this version?

Reply:
renovate I mean

Reply:
hopefully shouldn't need this PR as PR-506 should fix it but if we do, can look at renovate changes to prevent this upgrade